src/utils/types.ts

```diff
@@ -207,14 +230,27 @@ export type NonprofitUpdateNonprofit = Omit<
 >;
 export type NonprofitCreateIssue = Omit<
   Issue,
-  "_id" | "project" | "createdAt" | "updatedAt"
+  "_id" | "project" | "createdAt" | "updatedAt" | "finishedAt" | "reviewer"
 >;
 
 export type NonprofitUpdateIssue = Pick<
   Partial<Issue>,
   "status" | "finishedAt"
 >;
 
+export type ListQuery<
+  PARAMS extends Required<Omit<Record<string, unknown>, "filters">>,
```

Comment:
`PARAMS` are required parameters that need to be included in the query. It's possibly we want to move these to just required filters. I chose not to do this, however, because we currently use path parameters. This would lead to a redundancy of the parameters e.g.:
`/fetch-some-object/some-id/nonprofit?filters={id:some-id}`.

We can either keep it like it is (equired parameters not in filters), or we can remove our use of path parameters.

Reply:
I think it is fine to keep ids outside of filters.  Would anything else be considered a required param?